### ISSUE TITLE
Cannot update a plugin from a third party hub

### ISSUE THREAD

**Issue by francois2metz** [turbot/steampipe#2152]
**Describe the bug**
I have the [baleen](https://github.com/francois2metz/steampipe-plugin-baleen) installed from ghcr.io/francois2metz/baleen@latest. My last installed version is 0.0.4. When I type `steampipe plugin update ghcr.io/francois2metz/baleen`, I get `Error: there was an issue contacting the update server. Please try later.`

**Steampipe version (`steampipe -v`)**
steampipe version 0.14.6

**To reproduce**
```
steampipe plugin install ghcr.io/francois2metz/baleen
steampipe plugin update ghcr.io/francois2metz/baleen
```

**Expected behavior**
It should update the plugin

**Additional context**
STEAMPIPE_LOG=TRACE steampipe plugin update ghcr.io/francois2metz/baleen   
<details>
2022-06-07 12:40:16.479 UTC [TRACE] steampipe: Connection.PopulateChildren for aggregator connection scalingo
2022-06-07 12:40:16.479 UTC [TRACE] steampipe: Connection.PopulateChildren found matching connection scalingo_dev
2022-06-07 12:40:16.479 UTC [TRACE] steampipe: Connection.PopulateChildren found matching connection scalingo_prod

Error: there was an issue contacting the update server. Please try later.
</details>

**Comment by kaidaguerre** [comment]
@francois2metz I have reproduced this - it looks like the update code only currently works for turbot plugin

Looking into it.

**Comment by binaek** [comment]
Should be fixed by #3620
